### DISCUSSION TITLE
DNS PROBLEM?

### DISCUSSION THREAD

**Discussion by 404SynapseNotFound** [messense/aliyundrive-webdav#627] [Q&A]
stdout: [2m2022-10-22T03:00:20.512248Z[0m [33m WARN[0m [2maliyundrive_webdav::drive[0m[2m:[0m refresh token failed, will wait and retry [3merror[0m[2m=[0merror sending request for url (https://websv.aliyundrive.com/token/refresh): error trying to connect: dns error: failed to lookup address information: Name does not resolve

我用第三方网站ping websv.aliyundrive.com 也找不到主机

**Comment by jonathanfan-ee** [comment]
请问你解决了吗？我也遇到了这个问题。我在大陆阿里云服务器上无法连接https://websv.aliyundrive.com/token/refresh,报错相同。转本地机器不开代理同样无法连接。
查询结果如图
http://cdn.jonathanfan.top//1/uTools_1669729158144.png
